featurebyte/worker/task/feature_job_setting_analysis.py

```diff
@@ -100,10 +100,16 @@ async def execute(self, payload: FeatureJobSettingAnalysisTaskPayload) -> Any:
         database_type = feature_store.type
         if database_type in {SourceType.DATABRICKS, SourceType.DATABRICKS_UNITY}:
             database_type = SourceType.SPARK
+
+        table_details = DatabaseTableDetails(
+            database_name=event_table.tabular_source.table_details.database_name,
+            schema_name=event_table.tabular_source.table_details.schema_name,
+            table_name=event_table.tabular_source.table_details.table_name,
+        )
         event_dataset = EventDataset(
             database_type=database_type,
             event_table_name=event_table.name,
-            table_details=event_table.tabular_source.table_details,
+            table_details=table_details,
```

Comment:
Pushing a pydantic object to `featurebyte_freeware` causing wrong function pointer for `dict`. Creating the `featurebyte_freeware` dataclass before passing it to freeware